--- a/tests/toolbox.inc.sh
+++ b/tests/toolbox.inc.sh
@@ -17,10 +17,10 @@
 echo === $OUTPUTFILE ===
 
 endian=`file -L /proc/$$/exe`
-if expr "$endian" : '.* MSB \+\(executable\|shared object\).*' >&/dev/null
+if expr "$endian" : '.* MSB .*\(executable\|shared object\).*' >&/dev/null
 then
     endian=BE
-elif expr "$endian" : '.* LSB \+\(executable\|shared object\).*' >&/dev/null
+elif expr "$endian" : '.* LSB .*\(executable\|shared object\).*' >&/dev/null
 then
     endian=LE
 else
